src/xSTIR/cSTIR/stir_data_containers.h

```diff
@@ -549,9 +611,16 @@ class PETImageData : public aDataContainer<float> {
 	void get_voxel_sizes(float* vsizes) const;
 	int get_data(float* data) const;
 	int set_data(const float* data);
+	// GeometricalInfo get_patient_coord_geometrical_info() {
+	//   return *_patient_coord_geometrical_info;
+	// }
+	GeometricalInfo get_patient_coord_geometrical_info() const {
+		return *_patient_coord_geometrical_info;
+	}
 
 protected:
 	shared_ptr<Image3DF> _data;
+	shared_ptr<GeometricalInfo> _patient_coord_geometrical_info;
```

Comment:
not sure why we'd be using `shared_ptr` here.

Reply:
Ah.. Because @evgueni-ovtchinnikov told me too :s
It should be unique? Or no pointer?

Reply:
could be just GeometricalInfo as it is a very simple object that so far does not even seem to need destructor

Reply:
I just made it a little more complicated